backupwordpress.php

```diff
@@ -6,7 +6,7 @@
 Version: 3.3 Alpha
 Author: Human Made Limited
 Author URI: http://hmn.md/
-License: GPL-2.0+
+License: GPL-3.0
```

Comment:
`GPL-2.0+` - I assum means 2 and higher?

From https://wordpress.org/about/license/

> The license under which the WordPress software is released is the GPLv2 (or later)

I started looking at the differences but it gave me a headache.

I got a notice in Scrutinizer stating that it was not a valid SPDX license. So I checked out that page https://spdx.org/licenses/

I'm fine with reverting it back.